### PR TITLE
Added backend for enroll_student.php

### PR DESCRIPTION
**Pull request by adriitakumi**
Able to add to database, with the inclusion of form validation and 4 hours worth of blood, sweat, and tears. Ano pa babaguhin?

### CODE REVIEW

File: application/models/enroll_model.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	'fathername' => $data['fathername'],
+	'mothername' => $data['mothername'],
+	'fathercontact' => $data['fathercontact'],
+	'mothercontact' => $data['mothercontact'],
+	'guardian' => $data['guardian'],
+	'relationship' => $data['relationship'],
+	'guardiancontact' => $data['guardiancontact']
+	);
+	
+$this->db->insert('enroll', $studentInfo);
```
Comment (hakeempolistico) [inline application/models/enroll_model.php]: Try removing $studentInfo and retain `$this->db->insert('enroll', $data);`

File: application/models/enroll_model.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
```
Comment (hakeempolistico) [inline application/models/enroll_model.php]: Try removing $studentInfo and retain `$this->db->insert('enroll', $data);`

File: application/models/enroll_model.php (hunk 1 above)
Comment (hakeempolistico) [inline application/models/enroll_model.php]: Try removing $studentInfo and retain `$this->db->insert('enroll', $data);`

File: application/models/enroll_model.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	'barangay' => $data['barangay'],
+	'city' => $data['city'],
+	'province' => $data['province'],
+	'fathername' => $data['fathername'],
+	'mothername' => $data['mothername'],
+	'fathercontact' => $data['fathercontact'],
+	'mothercontact' => $data['mothercontact'],
+	'guardian' => $data['guardian'],
+	'relationship' => $data['relationship'],
+	'guardiancontact' => $data['guardiancontact']
```
Comment (hakeempolistico) [inline application/models/enroll_model.php]: sdfsaf

File: application/models/enroll_model.php [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+	);
```
Comment (hakeempolistico) [inline application/models/enroll_model.php]: sad

File: application/models/enroll_model.php (hunk 1 above)
Comment (adriitakumi) [inline application/models/enroll_model.php]: still works.

Comment (hakeempolistico) [inline application/models/enroll_model.php]: Good. Go commit.

**Pull request merged**